### PR TITLE
Update documentation on how to remove duplicate search results

### PR DESCRIPTION
**Pull request by MahmudH**
Search admin removes the result temporarily so a permanent solution is added to the docs

Trello: https://trello.com/c/DauneS2z

### CODE REVIEW

**Comment by cbaines** [comment]
Could you rephrase the first line of the commit message to say what the commit does:

"Removing duplicate search results updated"

as at the moment, it's a little difficult to guess what the commit is doing from reading the commit message.

File: source/manual/rename-a-country.html.md [hunk 1]
```diff
@@ -130,5 +130,6 @@ If the old country slug is still appearing in the search results, this can be re
 
 1. Navigate to the `Search results` page and enter the path to be removed
 2. Remove result, it should now disappear from the listing.
```
Comment (cbaines) [inline source/manual/rename-a-country.html.md]: Unless a specific reason for keeping the documentation regarding using the Search Admin is around, I'd suggest not mentioning it, and just mentioning the Whitehall approach which you're adding in.

Comment (MahmudH) [reply source/manual/rename-a-country.html.md]: I'm happy with that, I'll update it now 👍
